database/seeds/UsersTableSeeder.php

```diff
@@ -11,16 +11,14 @@ class UsersTableSeeder extends Seeder
      */
     public function run()
     {
-        $admin = BackpackUser::create([
+        $admin = User::create([
             'name' => 'Admin User',
             'email' => 'admin@'.preg_replace('/https?:\/\//', '', config('app.url')),
             'email_verified_at' => now(),
             'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
             'remember_token' => Str::random(10),
         ]);
 
-        $admin->assignRole(BackpackUser::ROLE_SUPER_ADMIN);
-
-        factory(BackpackUser::class, 5)->create();
+        $admin->assignRole(User::ROLE_SUPER_ADMIN);
```

Comment:
So this was due to the concept of [guards](https://spatie.be/docs/laravel-permission/v6/basic-usage/multiple-guards).

The roles were created using the `backpack` guard hard-coded in our `Role` model, whereas the checks were using the one default in our configuration file (`auth.defaults.guard`). I'm not sure why we override it in `Role`, and I don't know how it can even work in other situations (I'll check in production soon), so my fix only alters the seeder just to be safe.

Reply:
Yeah so it's the same in prod. I don't think we ever assign roles outside of the seeders anyway so it probably isn't an issue anywhere else. We could _probably_ just get rid of the hardcoded role in the model class, but I don't think we care enough about this app to risk anything.